cspell.json

```diff
@@ -10,27 +10,27 @@
 		"pnpm-lock.yaml"
 	],
 	"words": [
-		"Abhijit",
-		"Ajit",
 		"Astori",
-		"Blass",
 		"bluesky",
 		"bostonts",
-		"clsx",
-		"currentcolor",
-		"Dimitri",
-		"Farolino",
 		"Hadley",
-		"isbot",
+		// "isbot",
```

Comment:
What is this for?

Reply:
Oh right it was some internal thing used by Remix for the Philly JS Club site. I don't know if it's still needed.

Reply:
Welp, I guess it was indeed necessary: https://github.com/SquiggleTools/boston-ts-website/actions/runs/10874012817/job/30170816313?pr=340

Reply:
lol.